### ISSUE TITLE
[Accessibility] A11y_.Net Editor_Keyboard: Cannot be able to close the dialog box after pressing Developer : inspect tokens #102

### ISSUE THREAD

**Issue by LadyNaggaga** [Microsoft/monaco-editor#1162]
.NET Online REPL  is having the following accessibility issue.  Please see this [issue](https://github.com/dotnet/try/issues/102) . I have also included the details below 
> **Environment Details:**
> https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
> **Browser:** Edge (41.16299.402.0)
> Chrome (67.0.3396.87)
> 
> **Additional Details:**
> MAS Violated: MAS2.1.1
> 
> **Repro steps:**
> 
> 1. Hit the URL https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
> 2. Tab till “Enter focus mode” and press enter
> 3. Type any code and press F1 and tab till Developer : inspect tokens and press enter
> 4. Verify Whether the user can be able to close the dialogue box or not.
> 
> **Actual Result:**
> Cannot be able to close the dialog box after pressing Developer : inspect tokens.
> 
> **Expected Result:**
> Dialogue box should be closed with the help of the keyboard
> 
> **User Impact:**
> Keyboard users find it difficult if the dialogue box is opened and user wants to close it so there should be a short key in closing the dialogue box
> 
> **MAS Reference:**
> [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B8492c4eb-c179-40ae-8777-cd044ed725a2%7D)
> 
> ![dialog box cannot be closed with the help of keyboard](https://user-images.githubusercontent.com/41109135/42618171-d7edee0a-85d1-11e8-8be6-7cb237a100c8.png)

**Comment by alexandrudima** [comment]
See https://github.com/Microsoft/monaco-editor/issues/1165#issuecomment-437894496
